tests/Model/Command/UserMergeTest.php

```diff
@@ -0,0 +1,45 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\Model\Command;
+
+use PHPUnit\Framework\TestCase;
+
+class UserMergeTest extends TestCase
```

Comment:
We extend "our" TestCase in other test... It has no added value right now, but we may do it here as well to make it unified.